nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -261,6 +292,11 @@ in
         PIDFile = "/run/slurmctld.pid";
         ExecReload = "${pkgs.coreutils}/bin/kill -HUP $MAINPID";
       };
+
+      preStart = ''
+        mkdir -p ${cfg.stateSaveLocation}
+        chown -R ${cfg.user}:root ${cfg.stateSaveLocation}
```

Comment:
Why is it necessary to use group `root` here? It would be more consistent to use group `slurm`, see comment above.

Reply:
resolved